extension/content/components/recipes/form/arguments/MessagingExperiment.js

```diff
@@ -0,0 +1,221 @@
+// @ts-nocheck
```

Comment:
Out of curiosity, did you add this as copy/paste, or because you ran into some trouble and didn't want to deal with it right now?

Reply:
the latter

Reply:
i added ts-nocheck to the files that the check complained about

Reply:
Ah, this points out an interesting thing. I ran the type checker on this to see what it was complaining about. It's reading the actual parameters of the functional component, and ignoring the proptypes. The function parameters make it look like `options` is required (since it doesn't have a default value), but the proptypes say it's not. I'm sure this is going to cause some confusion. Oh well. This is fine for now.